code/modules/mob/living/carbon/human/species_types/snail.dm

```diff
@@ -15,6 +15,7 @@
 		FACEHAIR
 	)
 	inherent_traits = list(
+		TRAIT_NO_SLIP_ALL, //SKYRAT EDIT - This Was Mistakenly Taken Out, I Think
```

Comment:
this isnt a skyrat edit, it exists upstream

Reply:
true